registry/registryprovider.go

```diff
@@ -231,24 +300,31 @@ func IsGithubShortPackageType(t string) bool {
 	return PackageRegistryMatcher.MatchString(t)
 }
 
+// IsGCSShortType returns whether a given type is a type description in a short format to GCS
+func IsGCSShortType(t string) bool {
+	return strings.HasPrefix(t, "gs://")
+}
+
 // GetDownloadURLs checks a type to see if it is either a short git hub url or a fully specified URL
 // and returns the URLs that should be used to fetch it. If the url is not fetchable (primitive type
 // for example), it returns an empty slice.
-func GetDownloadURLs(rp RegistryProvider, t string) ([]string, error) {
+func GetDownloadURLs(rp RegistryProvider, t string) ([]string, Registry, error) {
```

Comment:
Check that the assumptions here about format align with the assumptions made by the chart reference that @technosophos defined.